ci-operator/step-registry/ipi/conf/gcp/osdisk/disktype-compute/ipi-conf-gcp-osdisk-disktype-compute-commands.sh

```diff
@@ -65,12 +65,14 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 PATCH="${SHARED_DIR}/install-config-patch.yaml"
 
 # the OCP version supports disk type "pd-balanced"
-EXPECTED_OCP_VERSION="4.14"
+EXPECTED_OCP_VERSION1="4.14"
+# the OCP version supports disk type "hyperdisk-balanced"
```

Comment:
Sounds like we did not enable the new disk type, we do not need to update the `EXPECTED_OCP_VERSION` var name, right?